services/acquisition_control/app/api/acquisitioncontrol.py

```diff
@@ -0,0 +1,115 @@
+# Copyright (C) 2023, BRAIN-LINK UG (haftungsbeschränkt). All Rights Reserved.
+# SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-ScanHub-Commercial
+
+"""Acquisition control. Receives control cmd from ui and controls scans on devices."""
+# pylint: disable=no-name-in-module
+# pylint: disable=too-many-statements
+# python -m uvicorn acquisitioncontrol:app --reload
+
+import json
+import logging
+import random
+
+import httpx
+from fastapi import APIRouter
+from pydantic.json import pydantic_encoder
+
+from .models import Commands, DeviceTask, ParametrizedSequence, ScanJob, ScanStatus
+
+DEBUG_FLAG = False
```

Comment:
Do we still need this?

Reply:
@chdinh I think it was added by you. What is your opionion on that?

Reply:
You can see how it is used below. we can use this to not have the UI setup - are we able to trigger a scan without UI if we remove this?

I would actually suggest to intropduce a dedicated endpoint which can be used without UI instead of having the DEBUG flag.

Reply:
As I understand the code, the debug case only creates a random record ID and a sequence placeholder.
The information can also be obtained by passing a sequence ID to the scan job.
This is independent on the scanhub frontend and can be achieved by using the swagger UI which is required anyway to test the endpoint, right? 
This is why I would remove the debug case.

Reply:
I removed the debug case for now after short discussion with @jobehrens